open_spiel/integration_tests/api_test.py

```diff
@@ -751,3 +751,9 @@ def load_tests(loader, tests, pattern):  # pylint: disable=invalid-name,g-doc-ar
 
 if __name__ == "__main__":
   absltest.main()
+
+# TODO checks
```

Comment:
Please make these TODOs clearer. You can link to github issues if you want, or just write a longer description of each in-line here.